invoke/program.py

```diff
@@ -985,3 +997,23 @@ def print_columns(
             else:
                 print(spec.rstrip())
         print("")
+
+    def _possible_commands_msg(self, unknown_cmd: str):
+        try:
+            all_tasks = self.scoped_collection.task_names
+        except AttributeError:
+            all_tasks = {}
+
+        possible_cmds = list(all_tasks.keys())
+        suggestions = difflib.get_close_matches(
```

Comment:
I am also working on a bit more advanced version of the string matching, more suits to `namespace.command` situation.